src/contexts/WalletConnect/index.tsx

```
@@ -69,42 +111,219 @@ export const WalletConnectProvider = ({
+    if (!wcProvider.current) {
+      return;
+    }
+
+    const topic = wcProvider.current.session?.topic;
+    if (topic) {
+      await wcProvider.current.client.disconnect({
+        topic,
+        reason: getSdkError('USER_DISCONNECTED'),
+      });
+
+      delete wcProvider.current.session;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)